### DISCUSSION TITLE
Spring Boot 3.2 support

### DISCUSSION THREAD

**Discussion by ro0t** [Netflix/dgs-framework#1820] [Q&A] [answered]
Hey.

I'm in the process of moving my platform from Spring Boot 2.7 to 3.2. Having a bit hard time getting DGS to work, I've updates to the latest version.

I'm receiving this message on startup, which I've never seen before. As if the @DgsQuery annotations are not mapping my methods correctly. I didn't notice any breaking code changes in the docs so I assume it's just a setup issue.

```
2024-02-18T13:03:09.421Z  INFO 80845 --- [on(2)-127.0.0.1] o.s.b.a.g.GraphQlAutoConfiguration       : GraphQL schema inspection:
	Unmapped fields: {Query=[ ...ALL MY QUERIES AND MUTATIONS ARE LISTED HERE... ]}
	Unmapped registrations: {}
	Skipped types: []
```

And all queries and mutations I run are just returning null in the result field. Example:
`{
  "data": {
    "organization": null
  }
}`

Any idea why this is happening?

**Comment by GrantGochnauer** [comment, marked as answer]
Strange - I'm running DGS 8.3.1 using Spring Boot 3.2.2 without issue. No changes when I moved from Spring Boot 2.7. Are you running latest DGS and DGS codegen plugins?

**Reply by ro0t** [reply]
Yep I've tried running the latest version of both.  Probably some conflict with another package using graphql-java.

But thanks for confirming that you are on latest Spring Boot, then I know it's just a me problem

**Reply by stefan-kueenzi** [reply]
I've had the exact same problem and opened a bug for this:
https://github.com/Netflix/dgs-framework/issues/1885

Maybe it's because of this bug and there is a easy workaround
